src/frontend/components/Felleskomponenter/Feilside/FeilsideInnhold.tsx

```diff
@@ -1,16 +1,21 @@
 import React, { FC } from 'react';
 
-import { BodyShort, Heading, Link, List } from '@navikt/ds-react';
+import { BodyShort, Button, Heading, Link } from '@navikt/ds-react';
 
 import { useSpråk } from '../../../context/SpråkContext';
 import { LocaleType } from '../../../typer/common';
 
-import { GåTilBakeTilForrigeSideLenke } from './GåTilbakeTilForrigeSideLenke';
-import { LastInnSidenPåNyttLenke } from './LastInnSidenPåNyttLenke';
-
 export const FeilsideInnhold: FC = () => {
     const { valgtLocale } = useSpråk();
 
+    const lastInnSidenPåNytt = () => {
+        window.location.reload();
+    };
+
+    const gåTilbakeTilForrigeSide = () => {
+        history.back();
```

Comment:
Om man bruker `react-router-dom` tror jeg man burde bruke `const navigate = useNavigate();`, var i alle fall slik før

Mulig @hanneolsen har noe input her

Reply:
Vi pleide å bruke `useNavigate()`, men ettersom `Router` provider'en ikke alltid kan brukes som wrapper for `Feilside`, vil det ikke fungere. Derfor bruker vi `history.back()` og `window.location.reload()` fremfor `useNavigate()`.

Reply:
Jepp, denne feilsiden er ikke wrappet i noen av de riktige kontekstene for at vi kan bruke `useNavigate`. Dermed må vi gå for old school varianten med `history` og `location` i stedet 😄